src/components/AreaAvatar/AreaAvatar.js

```diff
@@ -0,0 +1,51 @@
+import React, {useCallback, useState} from 'react';
+import {Avatar} from '../../Avatar';
+
+import styles from './AreaAvatar.less';
+
+export function AreaAvatar(props) {
+    let {_id, avatar, onClickAvatar, onClickCross, crossOnMouseEnter} = props;
+
+    var classNames = require('classnames/bind');
+    var cx = classNames.bind(styles);
+
+    var avatarClass = cx({
+        cursorPointer: onClickAvatar
+    }, styles.avatar)
```

Comment:
what's going on here? why do you use such weird construction?

Reply:
you could just
import cn from 'classNames';

const avatarClass = cn({ [styles.test]: onClickAvatar });